src/Controller/UserActivateController.php

```diff
@@ -25,8 +26,8 @@ public function __construct(
 
     public function __invoke(Request $request): JsonResponse
     {
-        $body = json_decode($request->getContent(), null, 512, JSON_THROW_ON_ERROR);
-        $activationCode = $body->activationCode;
+        $body = $request->toArray();
+        $activationCode = $body['activationCode'];
```

Comment:
You should handle that `$body['activationCode']` may not be set and report an error in that case, e.g.

```suggestion
        $activationCode = $body['activationCode'] ?? null;
        if (null === $activationCode) {
            throw new BadRequestHttpException('Missing activation code');
        }
```